CHANGELOG.md

```diff
@@ -24,6 +21,7 @@
 * [BUGFIX] Scrape: Resolve conflicts between multiple exported label prefixes. #9479 #9518
 * [BUGFIX] Scrape: Restart scrape loops when `__scrape_interval__` is changed. #9551
 * [BUGFIX] TSDB: Fix memory leak in samples deletion. #9151
+* [BUGFIX] UI: Fix assets loading when a prefix is used in `--web.route-prefix` or `--web.external-url`. #9586
```

Comment:
between 2.31 and 2.30, this bug doesn't exist right ? It was introduced (a bit by my fault) during the development of the 2.31.

So from my point of view it shouldn't appear in the final release note of the 2.31. But I guess it depends if the release note is the list of the change with the previous version, or with the previous release candidate version.

Reply:
You are correct, I have removed it. It should not appear.